### PR TITLE
[svgo] Add namespace to remove ugly require-import

### PR DESCRIPTION
**Pull request by ltetzlaff**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/5f4d4c1865c775704a415d3cb7aa5839e050b5ce/types/svgo/svgo-tests.ts#L1
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

Minor change, nothing added, just enables working with the definitions.

### CODE REVIEW

**Comment by bradleyayers** [comment]
Have you considered enabling `esModuleInterop` (as suggested in https://github.com/Microsoft/TypeScript/issues/5073#issuecomment-375918149)? It seems desirable to avoid "the hack" of adding an empty namespace now that the blessed `esModuleInterop` option is available.

Would that work for you?

**Comment by ltetzlaff** [comment]
Thanks for pointing on that, I wasn't aware of it.
Setting those flags would still let us drop the require then 👍

File: types/svgo/tsconfig.json [hunk 1]
```diff
@@ -14,10 +14,12 @@
         ],
         "types": [],
         "noEmit": true,
-        "forceConsistentCasingInFileNames": true
+        "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true,
+        "allowSyntheticDefaultImports": true
```
Comment (bradleyayers) [inline types/svgo/tsconfig.json]: Is this actually required? Other modules just have `esModuleInterop` e.g. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/67ff0213632be98f732e9c95d3812d069affe174/types/lodash/scripts/tsconfig.json#L6

**Comment by bradleyayers** [comment]
Technically this flag only needs to be set by a consumer (i.e. not in this repo). However setting it in this repo allows the tests file to use the concise import, and may serve as an example to consumers.

**Review by weswigham — CHANGES_REQUESTED** [review]
I'd appreciate it if you could do like @bradleyayers 's comment implied and remove the unneeded `allowSyntheticDefaultImports` flag; `esModuleInterop` on its own should do.

File: types/svgo/tsconfig.json (hunk 1 above)
Comment (ltetzlaff) [reply types/svgo/tsconfig.json]: It is indeed sufficient for TS 2.7+, but the lint set includes running tests for TS 2.6, where I ran into

```
ERROR: 1:8    expect  Compile error in typescript@2.6 but not in typescript@2.7.
Fix with a comment '// TypeScript Version: 2.7' just under the header.
Module '".../DefinitelyTyped/types/svgo/index"' has no default export.
```

thus added the flag.

Comment (weswigham) [reply types/svgo/tsconfig.json]: Hm. OK. `allowSyntheticDefaultImports` alone is also sufficient, so you can remove `esModuleInterop`. :P

**Comment by ltetzlaff** [comment]
@weswigham this should finally resolve it 😃🙏

**Comment by ltetzlaff** [comment]
I opened an issue for that bug in `types-publisher` leading to the failed travis build over at https://github.com/Microsoft/types-publisher/issues/454 and moved the comments to a `readme.md`.

**Pull request merged**